lib/splash.js

```diff
@@ -3,6 +3,8 @@ var _ = require("lodash");
 var level = 0;
 var _logger;
 
+var KEYS_TO_HIDE = ["pass", "secret"];
```

Comment:
@sebz, yes it could be nice to move this array to `node-tech-logger` configuration.